### PR TITLE
feat: adapt the contact creation page to new style

### PR DESCRIPTION
**Pull request by j-a-m-l**
## Proposed changes
This PR introduces the new style of the contact new page.

## Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)

## Checklist
- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

File: src/renderer/assets/images/pages/contact-new/purse.svg [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (alexbarnsley) [inline src/renderer/assets/images/pages/contact-new/purse.svg]: Can we call this `wallet.svg` instead

Comment (boldninja) [reply src/renderer/assets/images/pages/contact-new/purse.svg]: Smells like @olejegcord in here.

Comment (alexbarnsley) [reply src/renderer/assets/images/pages/contact-new/purse.svg]: image

Comment (alexbarnsley) [reply src/renderer/assets/images/pages/contact-new/purse.svg]: image

Comment (j-a-m-l) [reply src/renderer/assets/images/pages/contact-new/purse.svg]: The initial file that I got was called `images.svg` or something alike, so I pay my own tribute to Oleg, calling it `purse.svg` to make it less confusing than calling it `wallet.svg`, which could be interpreted as a background for wallets, not for contacts only.

I've renamed it to `wallets.svg`, @alexbarnsley .

**Pull request merged**